source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRChoiceSetViewDataSource.mm

```diff
@@ -19,7 +19,8 @@
 NSString *checkedRadioButtonReuseID = @"checked-radiobutton";
 NSString *uncheckedRadioButtonReuseID = @"unchecked-radiobutton";
 
-const CGFloat padding = 2.0f;
+const CGFloat padding = 8.0f;
```

Comment:
Do we need to add this new expectation in extenal facing docs @jwoo-msft  ?

Reply:
@VikrantSingh01 
We could add a note about that to the release notes.

Reply:
@GabrielMedAlv : fyi ^

Reply:
I plan to add these as part of release notes:

- Increased height of choice set cell to 44 pixels
- Increased separation between choice icon and text to 8 pixels.